### ISSUE TITLE
Release a version for Scala 2.12.0-M1

### ISSUE THREAD

**Issue by julien-truffaut** [scalamacros/paradise#70]
I am not sure of what is the long term plan for this project, maybe it will be integrated into 2.12 directly.

**Comment by xeno-by** [comment]
Hi @julien-truffaut! Sorry for the late answer - apparently this notification got buried under a pile of other github notifications, and when I marked everything as read, I didn't take a close enough look at everything.

This project won't be integrated in 2.12, but I can and will maintain it for 2.12. How urgent is the 2.12.0-M1 release for you?

**Comment by julien-truffaut** [comment]
np, it is not urgent but I think it is the last dependency that blocks me from building Monocle for 2.12.0-M1.

**Comment by julien-truffaut** [comment]
@xeno-by any idea when do you think it can be done?

**Comment by xeno-by** [comment]
@julien-truffaut Thanks for reminding. I'll take care of it this week.

**Comment by xeno-by** [comment]
Just released 2.1.0-M5 for both 2.12.0-M1 and 2.12.0-M2.

**Comment by julien-truffaut** [comment]
I still get: 

```
unresolved dependency: org.scalamacros#paradise_2.12.0-M1;2.0.1: not found
```

but maybe I am missing something.

**Comment by xeno-by** [comment]
Should be fixed now. Also released 2.0.1.
